### PR TITLE
[FIX] don't use patch_method in tests, this confuses overrides

### PR DESCRIPTION
**Pull request by hbrunn**
the test code for example destroys `server_environment_ir_config_parameter` because the patching restores the original function, not the overriden one

### CODE REVIEW

File: auth_session_timeout/tests/test_ir_config_parameter.py [hunk 1]
```diff
@@ -30,47 +30,42 @@ class TestIrConfigParameterCaching(common.TransactionCase):
...
+        """First call should actually run the function."""
+        cache = self._cache('_auth_timeout_get_parameter_delay')
+        cache_misses = cache.miss
         self.param_obj._auth_timeout_get_parameter_delay()
-        self.assertTrue(self.get_param_called)
+        self.assertEqual(cache.miss, cache_misses + 1)
 
     def test_auth_timeout_get_parameter_ignored_urls_cache(self):
+        """First call should actually run the function."""
         """It should cache the parameter call."""
```
Comment (NL66278) [inline auth_session_timeout/tests/test_ir_config_parameter.py]: This line is causing the travis error. You cannot have a second doc string.

**Comment by NL66278** [comment]
Just wondering wether it would not be better to just backport the 11.0 version of auth_session_timeout, as suggested in the README of this module, as the current setup is apparently not working since commit da1f153 to Odoo. Well anyway, with the one line fixed that is making Travis red, this at least should make tests working again.

**Review by NL66278 — CHANGES_REQUESTED** [review]
Looks good, but very small change needed.

**Comment by hbrunn** [comment]
thanks, done. The backport should happen indeed, but is out of scope for now

**Comment by NL66278** [comment]
@hbrunn Maybe we can have a look at this one... https://github.com/OCA/server-tools/pull/1540/files

**Comment by NL66278** [comment]
Anyway changed method still using the same ir_config_parameter changes and tests, so the backport and this PR are both needed.

**Review by NL66278 — APPROVED** [review]
:+1:  LGTM. Apparant errors are due to travis failing.

**Review by gurneyalex — APPROVED** [review]

**Pull request merged**